chart/ory/templates/psp.yaml

```diff
@@ -1,34 +0,0 @@
-{{- if .Values.maintenance.podSecurityPolicy.enabled }}
```

Comment:
We removed the file but seems we still have the chart configuration `maintenance -> podSecurityPolicy`?